styles/website.css

```
@@ -38,6 +38,57 @@ body {
   overflow: visible;
 }
 
+/* In page TOC */
+.intopic-toc h3 {
+  color: #333;
+  font-size: 1.1em;
+}
+
+/* Parameter table */
+.api_input + table {
+  width: 100%;
+}
+
+/* Name */
+.api_input + table tr th:nth-child(1) {
```

Remove `tr`. Check code below, it also have same issues
```suggestion
.api_input + table th:nth-child(1) {
```